docker/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM postgres:14.6 as base
+FROM postgres:14.9-bullseye as base
 
 # postgres sub-package versions are  nailed here is to ensure we build dev against the same version as installed
 # the postgres vesions on http://apt.postgresql.org/pub/repos/apt/ might have changed since the last build
 # if this version is not available anymore we need to use a different base image which btw might also change
 # the debian distribution
-ENV PG_APT_VERSION="14.6-1.pgdg110+1"
+ENV PG_APT_VERSION="14.9-1.pgdg110+1"
```

Comment:
`14.6-1.pgdg110+1` version was not available anymore, so I could not build an image locally. based on the comment above, I've found this version. Since it starts with 14.9, I updated the base image too, but `postgres:14.9` image is now based on Debian Bookworm, so I used `postgres:14.9-bullseye` to keep the same distro.